control/freqplot.py

```diff
@@ -380,26 +424,55 @@ def default_frequency_range(syslist):
     for sys in syslist:
         try:
             # Add new features to the list
-            features = np.concatenate((features, np.abs(sys.pole())))
-            features = np.concatenate((features, np.abs(sys.zero())))
+            if sys.isctime():
+                features_ = np.abs(sys.pole())
+                features_ = np.concatenate((features_, np.abs(sys.zero())))                
+                # Get rid of poles and zeros at the origin
+                features_ = features_[features_ != 0.0];
+                features = np.concatenate((features, features_))
+            elif sys.isdtime(strict=True):
+                fn = np.pi * 1. / sys.dt
+                #TODO: What distance to the Nyquist frequency is appropriate?                
+                freq_interesting.append(fn*0.9)
+                p = sys.pole()
+                p = p[p != -1.]
+                features = np.concatenate((features, np.abs(np.log(p) / sys.dt)))
```

Comment:
p != -1 lets through poles at 0, which will give an inf in the np.log(p) call.  It also doesn't correspond to the comment below ("Get rid of poles and zeros ...")